get-booking/src/bookings.json

```diff
@@ -2,15 +2,15 @@
   "version": "1",
   "bookings": [{
     "bookingId": "AABBCC1234567",
-    "dln": "32519228",
+    "dln": "MORGA657054SM9IJ",
     "bookingDate": "14/03/18"
```

Comment:
bookings.json should only contain the DLNs.

If 'get-booking' receives a request for a DLN it knows, invent a new booking and return it.
A real 'get-booking' would want to interpret the date it received; our stub version does not.

For 'bookingId', again, don't have it in the bookings.json .
For this stub, do something meaningless but repeatable, e.g. calculate the MD5 of the DLN and use that.

Reply:
Done